sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/second/SecondScreen.kt

```diff
@@ -1,27 +1,40 @@
 package co.nimblehq.sample.compose.ui.screens.second
 
-import androidx.compose.foundation.layout.Box
-import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.*
 import androidx.compose.material.Scaffold
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.hilt.navigation.compose.hiltViewModel
 import co.nimblehq.sample.compose.R
-import co.nimblehq.sample.compose.model.UiModel
+import co.nimblehq.sample.compose.ui.AppDestination
 import co.nimblehq.sample.compose.ui.screens.AppBar
-import co.nimblehq.sample.compose.ui.theme.ComposeSampleTheme
+import co.nimblehq.sample.compose.ui.theme.ComposeTheme
 
 @Composable
-fun SecondScreen(uiModel: UiModel) {
+fun SecondScreen(
+    viewModel: SecondViewModel = hiltViewModel(),
+    navigator: (destination: AppDestination) -> Unit,
```

Comment:
@Tuubz But we have `SecondViewMode` in `sample-xml`; it seems like we missed some process to emit the value to `VM` for `sample-compose`. I think we should keep it because after we finish setup the modules, then we will have more examples about `Compose` 🤔 cc @luongvo

Reply:
Hmmm, good catch @Wadeewee  👍 

@luongvo Would it make sense to also collect the `id` from `VM` for `sample-compose`? 🙏

Reply:
@Tuubz Can you clarify your suggestion a bit clearer? 🙏

Reply:
**Conclusion**: https://nimble-co.slack.com/archives/G41UCTGHL/p1672136329771999?thread_ts=1672109944.295459&cid=G41UCTGHL